# pytest-runner should be conditional on test arg provided to setup.py
# it is only needed for tests, but we run pytest directly (for now)
# TODO: Report/resolve upstream

--- setup.cfg.orig	2022-11-16 20:31:48 UTC
+++ setup.cfg
@@ -52,7 +52,6 @@ test = 
 	sphinx
 	sphinx_rtd_theme
 test = 
-	pytest
 	flake8
 	isort
 	numpy
